metagpt/utils/token_counter.py

```diff
@@ -128,11 +128,15 @@
 Different model has different detail page. Attention, some model are free for a limited time.
 """
 DASHSCOPE_TOKEN_COSTS = {
-    "qwen-turbo": {"prompt": 0.0011, "completion": 0.0011},
-    "qwen-plus": {"prompt": 0.0028, "completion": 0.0028},
-    "qwen-max": {"prompt": 0.0, "completion": 0.0},
-    "qwen-max-1201": {"prompt": 0.0, "completion": 0.0},
-    "qwen-max-longcontext": {"prompt": 0.0, "completion": 0.0},
+    "qwen-turbo": {"prompt": 0.00028, "completion": 0.00083},
+    "qwen-long": {"prompt": 0.00007, "completion": 0.00028},
+    "qwen-plus": {"prompt": 0.00055, "completion": 0.00166},
+    "qwen-max": {"prompt": 0.0055, "completion": 0.0166},
+    "qwen-max-0428": {"prompt": 0.0055, "completion": 0.0166},
+    "qwen-max-0403": {"prompt": 0.0055, "completion": 0.0166},
+    "qwen-max-0107": {"prompt": 0.0055, "completion": 0.0166},
+    "qwen-max-1201": {"prompt": 0.0166, "completion": 0.0166},
+    "qwen-max-longcontext": {"prompt": 0.0055, "completion": 0.0166},
```

Comment:
It seems that the token cost of these models should be updated together with the length supported by the model.